### PR TITLE
[DO NOT MERGE] feat: add theme v2 to health survey template

### PR DESCRIPTION
**Pull request by manoldonev**
TODO:
- [x] Theme: theme variables to be imported via @import "~@nativescript/theme/css/core.css";
- [ ] Theme: Remove default font-weight setting in ns-root (src/scss/core/primitives/_index.scss) 
- [x] Theme: Move Button default padding to core theme (padding: 0 5;)
- [ ] RadDataForm with TKPropertyGroup not styled correctly with iOS dark mode
- [ ] DataFormEditorLabel not aware of light/dark mode

Will not merge this for {N} 6.2.0 -- temporary workaround hardcoding light mode here: https://github.com/NativeScript/nativescript-app-templates/pull/114

### CODE REVIEW

File: packages/template-health-survey-ng/package.json [hunk 1]
```diff
@@ -50,19 +50,19 @@
     "kinvey-nativescript-sdk": "4.2.3",
     "nativescript-angular": "~8.2.0",
     "nativescript-iqkeyboardmanager": "~1.5.0",
-    "nativescript-theme-core": "~1.0.6",
-    "nativescript-ui-dataform": "~5.0.0",
+    "@nativescript/theme": "~2.1.0",
```
Comment (vchimev) [inline packages/template-health-survey-ng/package.json]: Order alphabetically for easier maintenance - scoped packages first.

File: packages/template-health-survey-ng/src/app.css [hunk 2]
```diff
@@ -3,39 +3,62 @@ In NativeScript, the app.css file is where you place CSS rules that
...
 http://docs.nativescript.org/ui/styling for a full list of the CSS
 selectors and properties you can use to style UI components.
-
 /*
 In many cases you may want to use the NativeScript core theme instead
-of writing your own CSS rules. For a full list of class names in the theme
-refer to http://docs.nativescript.org/ui/theme.
+of writing your own CSS rules. You can learn more about the 
+NativeScript core theme at https://github.com/nativescript/theme
+The imported CSS rules must precede all other types of rules.
```
Comment (MartoYankov) [inline packages/template-health-survey-ng/src/app.css]: I'm not sure we should teach people CSS. In this case this is not simply good practice - imports that are not on top are invalid CSS.

File: packages/template-health-survey-ng/src/app.css [hunk 3]
```diff
@@ -3,39 +3,62 @@ In NativeScript, the app.css file is where you place CSS rules that
...
-}
\ No newline at end of file
+}
+
+DataFormEditorCore {
+    border-width: 0;
+}
+
+DataFormEditorLabel { 
+    color:var(--light-text-color);
```
Comment (MartoYankov) [inline packages/template-health-survey-ng/src/app.css]:
```suggestion
    color: var(--light-text-color);
```

File: packages/template-health-survey-ng/src/app/consent/consent-review/review/review.component.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
-<ActionBar class="action-bar">
-    <Label class="action-bar-title" text="Consent Step 8 of 9" class="font-weight-bold"></Label>
+<ActionBar>
+    <Label text="Consent Step 8 of 9" class="font-weight-bold"></Label>
     <ActionItem (tap)="appService.logOut()" ios.position="right">
-        <Button text="Cancel" class="action-item text-center text-primary"></Button>
+        <Button text="Cancel" class="text-primary"></Button>
```
Comment (MartoYankov) [inline packages/template-health-survey-ng/src/app/consent/consent-review/review/review.component.html]: Shouldn't we leave the `text-center` class?

File: packages/template-health-survey-ng/src/app.css [hunk 5]
```diff
@@ -3,39 +3,62 @@ In NativeScript, the app.css file is where you place CSS rules that
...
+    font-weight: bold; /* TODO: remove this when default font-weight is removed from ns-root */
+}
+
+.ns-dark DataFormEditorLabel {
```
Comment (MartoYankov) [inline packages/template-health-survey-ng/src/app.css]: Shouldn't this be part of the theme too? I see there are styles about the forms plugin.

Comment (manoldonev) [reply packages/template-health-survey-ng/src/app.css]: It should be -- there is a todo item above `DataFormEditorLabel not aware of light/dark mode`, generally there is more work on RadDataForm styling so we decided to not release the rework with {N} 6.2.0

File: packages/template-health-survey-ng/src/app/consent/consent-review/review/review.component.html (hunk 4 above)
Comment (manoldonev) [reply packages/template-health-survey-ng/src/app/consent/consent-review/review/review.component.html]: It does nothing in this case so I removed it.

**Pull request closed without merging**